rpmlb/builder/koji.py

```diff
@@ -44,7 +44,10 @@ def __init__(
         self.epel = koji_epel
 
         #: Format string for the build target
-        self.target_format = koji_target_format
+        if koji_target_format is None:
+            self.target_format = type(self).DEFAULT_TARGET_FORMAT
```

Comment:
Oh. You should not assign to class level variable using self, because that would override it for the given object. But reading is fine.